modular_skyrat/modules/cortical_borer/code/cortical_borer.dm

```diff
@@ -289,12 +294,12 @@ GLOBAL_VAR_INIT(successful_blood_chem, 0)
 
 /mob/living/simple_animal/cortical_borer/Life(delta_time, times_fired)
 	. = ..()
-	//can only do stuff when we are inside a human
-	if(!inside_human())
+	//can only do stuff when we are inside a LIVING human
+	if(!inside_human() || (human_host.stat == DEAD))
 		return
 
 	//there needs to be a negative to having a borer
-	if(prob(5) && human_host.getToxLoss() <= 20)
+	if(prob(5) && human_host.getToxLoss() <= 80)
```

Comment:
this is brutal because remember, whenever someone has a borer, they are unable to see their health (similar to miner salve).